core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -324,8 +328,21 @@ protected void doStop() throws Exception {
     StatisticsHandler statsHandler = new StatisticsHandler();
     statsHandler.setHandler(handlers);
 
-    server.setHandler(statsHandler);
-
+    final ServletContextHandler webSocketServletContext =
+        new ServletContextHandler(ServletContextHandler.SESSIONS);
+    webSocketServletContext.setContextPath(
+        config.getString(RestConfig.WEBSOCKET_PATH_PREFIX_CONFIG)
+    );
+    final ContextHandlerCollection contexts = new ContextHandlerCollection();
+    contexts.setHandlers(new Handler[] {
+        webSocketServletContext,
```

Comment:
I wonder if we should just skip adding the websockets endpoint altogether if it isn't defined. This way we don't add it unless explicitly requested by the app?

Reply:
the only thing i like more about this is that it will make the default for anything we use more standard. e.g. if some other product wants to use sockets it would by default be under the same location

Reply:
ok, that's a good argument